tests/cmdline/commands/test_computer.py

```diff
@@ -324,6 +371,7 @@ def setUp(self):
         self.comp_builder.prepend_text = ''
         self.comp_builder.append_text = ''
         self.comp_builder.mpiprocs_per_machine = 8
+        self.comp_builder.default_memory_per_machine = 0
```

Comment:
```suggestion
```

Reply:
I don't think this should be removed. Same as mpiprocs_per_machine - you can't remove them here.